test/verify/check-journal

```diff
@@ -414,7 +415,8 @@ ExecStart=/bin/sh -c 'for s in $(seq 10); do echo SLOW; sleep 0.1; done; sleep 1
         b.wait_present(sel)
 
     @skipImage("ABRT not available", "debian-stable", "debian-testing", "ubuntu-stable",
-               "ubuntu-1804", "ubuntu-2004", "fedora-coreos", "rhel-8-2", "rhel-8-2-distropkg", "centos-8-stream")
+               "ubuntu-1804", "ubuntu-2004", "fedora-coreos", "rhel-8-2", "rhel-8-2-distropkg",
+               "centos-8-stream", "rhel-8-3")
```

Comment:
Nitpick: Would be nicer to keep all the RHELs together.

Reply:
Done